chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,10 @@
   EOH
 end
 
-# XXX requires chef-client with CHEF-4090 fixed otherwise the package
-# provider can't handle the URL
-package "https://#{manager_server}/pub/bootstrap/sm-client-tools.rpm"
-
-execute "sm-client" do
-  command "sm-client --hostname #{manager_server} --activation-keys #{activation_key}"
+bootstap_script = "bootstrap-sles12.#{node[:platform_version].split(".").last}"
+execute "bootstrap SUMA client" do
+  command "curl https://#{manager_server}/pub/bootstrap/#{bootstap_script}.sh | sh"
```

Comment:
have you tried the script yet? my question is only if it needs higher permission or anything. difficult to say when you can't see what it does